terraform/helm/genesis/templates/genesis.yaml

```diff
@@ -31,6 +31,9 @@ data:
     {{- with .Values.chain.on_chain_execution_config}}
     on_chain_execution_config: {{ . | toJson }}
     {{- end}}
+    {{- with .Values.chain.jwk_consensus_config_override }}
+    jwk_consensus_config_override: {{ . | toJson }}
+    {{- end}}
```

Comment:
@ibalajiarun what's a good way to test this?

Reply:
You can test this in Forge. Temporarily, modify the `realistic_env_max_load_test` with the override values and check if they get enabled. If you have transactions to test this in txn-emitter, then you can also use them. Or, if you have logs, you can manually check.

Reply:
See this for example: https://github.com/aptos-labs/aptos-core/blob/eab3696cc4edb42353178f29f56162ed9be558fa/testsuite/forge-cli/src/main.rs#L1941